Improbable/Stdlib/Improbable.Stdlib/Ops.cs

```diff
@@ -99,6 +102,7 @@ IEnumerator IEnumerable.GetEnumerator()
             return GetEnumerator();
         }
 
+        [DebuggerDisplay("{"+ nameof(Worker.CInterop.OpType) + "}")]
```

Comment:
Shouldn't this be the instance `OpType` rather than just the class enum name?

Reply:
So, this evaluates to the name of the instance field, so at runtime it shows the value of that variable (That's what the surrounding "{}" do.)
I just use `nameof` to avoid refactoring woes when/if the variable changes and the debuggerstring quietly breaks because it's hard-coded into a string literal

Reply:
That's so magic ✨